tests/backup/backup_basic_test.go

```diff
@@ -2,11 +2,6 @@ package tests
 
 import (
 	"fmt"
-	"os"
-	"strings"
```

Comment:
Why are we moving it down?

Reply:
This is Goland's way for formatting.

Reply:
But ideally standard libs should be on top right?

Reply:
I ran the go formatter again and it did not show any change in this file

Reply:
Okay but this is a concern, we would need to find out more here. I see that in the PRs it keeps moving it up and down.

Reply:
Yes. I agree. Even I am seeing this frequently.